README.md

```diff
@@ -1,15 +1,77 @@
 # Getting started with the Blazor Calendar Component
 
-A quick-start blazor project that lets you to get start with the Blazor Calendar component of Syncfusion and contains code snippets used to view and select dates. In this project, few of its important features are enabled like setting specific date range values to the calendar, multi-date selection, and navigation between different calendar views such as month, year and decade. 
+The [Blazor calendar](https://www.syncfusion.com/blazor-components/blazor-calendar?utm_source=github&utm_medium=listing&utm_campaign=blazor-calendars-github-samples)is a component to display the date and days of the week. A quick-start blazor project that lets you to get start with the Blazor Calendar component of Syncfusion and contains code snippets used to view and select dates. In this project, few of its important features are enabled like setting specific date range values to the calendar, multi-date selection, and navigation between different calendar views such as month, year and decade. 
 
-Documentation: https://blazor.syncfusion.com/documentation/calendar/getting-started/
-
-Online examples: https://blazor.syncfusion.com/demos/calendar/default-functionalities
- 
 ## Project prerequisites
 
 Make sure that you have the compatible versions of Visual Studio 2019 and .NET Core SDK 3.1.2 in your machine before starting to work on this project.
 
 ## How to run this application?
 
 To run this application, you need to first clone the `get-started-with-blazor-calendar` repository and then open it in Visual Studio 2019. Now, simply build and run your project to view the output.
+
+## Features and Benefits
+
+### Accessibility
+
+The web [accessibility](https://blazor.syncfusion.com/documentation/calendar/accessibility?utm_source=github&utm_medium=listing&utm_campaign=blazor-calendars-github-samples) makes web content and web applications more accessible for disabled people.It helps disabled persons by providing information about the widget for assistive technology in the screen readers.
+
+### Islamic Calendar
+
+In addition to the Gregorian calendar, the Blazor Calendar component supports displaying the [Islamic calendar](https://blazor.syncfusion.com/documentation/calendar/islamic-calendar) (Hijri calendar).
```

Comment:
add UTM param here and UTM should be 

?utm_source=github&utm_medium=listing&utm_campaign=blazor-calendar-github-samples

change this in all places